pipelines/managed/push-to-external-registry/push-to-external-registry.yaml

```diff
@@ -266,7 +349,48 @@ spec:
         - name: data
           workspace: release-workspace
       runAfter:
-        - verify-enterprise-contract
+        - collect-data
+    - name: create-pyxis-image
```

Comment:
Why add the pyxis tasks?

Reply:
+1. This pipeline is not supposed to do anything red hat specific - just verify conforma and push to quay.

Reply:
yep. bad copy paste. sorry. fixed